cli/azd/pkg/output/ux/endpoint.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type Endpoint struct {
-	Endpoint string `json:"Endpoint"`
+	Endpoint string
```

Comment:
@weikanglim - Perhaps we should make this a `[]string`?  I think with the APIM stuff we are moving towards a world where there might be multiple endpoints for a service?  So name this `endpoints` and have it be an array?

Reply:
The only place we use it is here:

```
		// report endpoint
		for _, endpoint := range svcDeploymentResult.Endpoints {
			d.console.MessageUxItem(ctx, &ux.Endpoint{Endpoint: endpoint})
		}
```

So this is handling the `[]string -> string` stuff at the layer where we emit the event.  I could go either way on which one we want, but I feel like I have a slight preference for "here's a single event that represents all the endpoints for a given deployed service".

Reply:
That's correct.